lib/manager/poetry/__snapshots__/extract.spec.ts.snap

```diff
@@ -115,6 +115,15 @@ Array [
       "nestedVersion": false,
     },
   },
+  Object {
+    "currentValue": "~2.7 || ^3.4",
```

Comment:
Do we need to specify versioning=poetry here?

Reply:
maybe it makes sense to set `poetry` versioning as default at extract phase. I think this should be done in a seperate pr

Reply:
ok